include/grpcpp/impl/codegen/client_callback.h

```diff
@@ -174,6 +174,8 @@ class ClientCallbackUnary {
 // StartWrite, or AddHold operations on the streaming object. Note that none of
 // the classes are pure; all reactions have a default empty reaction so that the
 // user class only needs to override those classes that it cares about.
+// The reactor must be passed to the stub invocation before any of the below
+// operations can be called.
```

Comment:
Isn't that redundant with the 2nd sentence of the paragraph? "They are passed in to the library as an argument to a call on a stub."

Reply:
The 2nd sentence is weaker IMO. When reading this for the first time, it wasn't clear to me that I couldn't call these functions before passing the reactor to the library. For someone who knows how these things work, it's probably obvious, but it may not be completely clear to someone who is writing callback API stuff for the first time.

Reply:
And when I say weaker, I mean a weaker guarantee / statement on semantics. Not weaker in terms of quality :)

Reply:
Yeah, I guess that makes sense.